src/database/data-source.ts

```diff
@@ -0,0 +1,19 @@
+import "dotenv/config"
+import "reflect-metadata";
+import { DataSource } from "typeorm";
+// import { client } from @/entities;
+// import { sample } from @/entities;
+// import { result } from @/entities;
+
+export const AppDataSource = new DataSource({
+    type: "postgres",
+    host: process.env.DB_HOST,
+    port: parseInt(process.env.DB_PORT || '5432'),
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    synchronize: false,
+    logging: true,
```

Comment:
make this dynamic. Have a config file (if you dont)..logging should only be true if node environment is not production

`entities: config.app.isTsNode
    ? ["src/**/*.entity.ts"]
    : ["dist/**/*.entity.js"],
  migrations: config.app.isTsNode
    ? ["src/database/migrations/*.ts"]
    : ["dist/database/migrations/*.js"],
  synchronize: false,
  migrationsRun: config.app.isTsNode,
  logging: config.app.isTsNode,`

I have **/* because of my folder structure. I'm also using istsnode because of the multiple environments I'm working with. yours wont neccessarily be the same